client/src/components/App/App.tsx

```diff
@@ -37,14 +37,16 @@ const App: React.FC = (): JSX.Element => {
     const [isUserUpdated, setIsUserUpdated] = useState<boolean>(true);
     const [isSignUpOpen, setIsSignUpOpen] = useState<boolean>(false);
 
+    const handleOpenSignUp = () => setIsSignUpOpen(true);
```

Comment:
I don't think you are gaining much from wrapping the `setIsSignUpOpen` with handle functions

Reply:
just to make it understandable

Reply:
I don't think `handleOpenSignUp` adds more readability than `setIsSignUpOpen(true)` in this scenario.

Reply:
fixed